handbook/rbac/rbac.md

```diff
@@ -0,0 +1,128 @@
+
+### Users and Roles
+
+Within each workspace user, systems, and analysis procedures are assigned roles. Role grant privileges that allow access to the various SystemLink plugins and the data exposed by those plugins. Privileges explicitly grant what you *can* do - there are no *deny* privileges. Due to this a user may be assigned multiple roles within a single workspace that collectively describe what the user has access to. This is useful to prevent the proliferation of multiple different roles that have similar privileges. Instead simple roles may be defined and be composed together by assigning a user each role.
+
+!!! note "Allow all privileges"
+
+    In each area of available privileges there exists a special privilege: **Allow all privileges**. This privilege automatically grants all privileges for that area. New privileges may be added to new release of SystemLink. If a new privilege is added to the area upon upgrade users with the **Allow all privileges** privilege will automatically be granted the new privilege. If this behavior is undesirable, refrain from using **Allow all privileges**.
+
+    <figure>
+        <img src="../../img/allow-all-privileges.png" width="500" />
+        <figcaption>All privileges for an area are granted with Allow all privileges is checked</figcaption>
+    </figure>
+
+#### Built-in and Custom Roles
+
+Roles define the privileges a user may take against resources within SystemLink. While a user has a role within the context of a workspace a role is defined globally. This enables the same role to be used across multiple workspaces.
+
+When creating an assigning roles, it is best practice to provide users with the minimum number of privileges to perform their work. SystemLink has granular privileges for services and application in the product. These privileges also include access to individual SystemLink applications such as **Systems Manager** and **Test Monitor**.
+
+!!! note
+    If a user has a role that grants access to an application in any workspace that application will be available to the user at all times. If a user has not been granted access to an application in any workspace that application will be hidden from the navigation menu.
+
+Custom roles may be created, duplicated, modified, and deleted as needed. If a role is updated all users mapped to that role are affected.
+
+Due to the large number of privileges available SystemLink includes several *built-in* roles to facilitate getting started. Go to the SystemLink **Security** application to review the details on privileges granted for each role.
+
+- **Systems Maintainer**: This role has full access to systems, packages, assets, alarms, dashboards, and WebVIs. This role had read-only access to data and analysis procedures.
```

Comment:
```suggestion
- **Systems Maintainer**: This role has full access to systems, packages, assets, alarms, dashboards, and WebVIs. This role has read-only access to data and analysis procedures.
```

Reply:
I decided to remove this section and instead reference the manual on ni.com